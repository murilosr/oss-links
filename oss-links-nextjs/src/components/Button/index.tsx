import React from "react";

export default function Button(props: {text: string}){
    return (
        <div className="p-6">{props.text}</div>
    );
}