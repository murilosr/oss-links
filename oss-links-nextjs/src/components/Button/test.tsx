import { render, screen } from '@testing-library/react'
import Button from 'src/components/Button'
import '@testing-library/jest-dom'

describe('Home', () => {

  it('renders a div', () => {
    const {container} = render(<Button text="Hello World ^^"/>);

    const btn = container.querySelector('div');

    expect(btn).toMatchSnapshot();
  })

  it('renders a div 2', () => {
    const {container} = render(<Button text="This is a button ^^"/>);
    const btn = container.querySelector('div');
    expect(btn).toMatchSnapshot()
  })
  
})