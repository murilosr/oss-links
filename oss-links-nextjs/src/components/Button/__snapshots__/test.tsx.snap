// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders a div 1`] = `
<div
  class="p-6"
>
  Hello World ^^
</div>
`;

exports[`Home renders a div 2 1`] = `
<div
  class="p-6"
>
  This is a button ^^
</div>
`;
